src/fluree/db/connection.cljc

```diff
@@ -434,6 +432,25 @@
   (remove-listener* (:state conn) network dbid key))
 
 
+
+(defn add-token
+  "Adds token to connection information so it is available to submit storage read requests.
+
+  Returns true if successful, false otherwise."
+  [conn token]
+  (let [conn-id (:id conn)
+        _ (log/info "add-token" {:conn-id conn-id :token token})
+        ]
+    (try
+      (swap! server-connections-atom update-in [conn-id :token]
```

Comment:
Can use: (swap! server-connections-atom update-in [conn-id :token] #(or % token))

Alternatively if there could a boolean "false" value the above would overwrite it, but alternative if that is a concern could be:
(swap! server-connections-atom update-in [conn-id :token] (fnil identity token))